kloppy/tests/test_pff.py

```diff
@@ -0,0 +1,210 @@
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+
+import pytest
+
+from kloppy.domain import (
+    Provider,
+    Orientation,
+    Point,
+    Point3D,
+    DatasetType,
+)
+
+from kloppy import pff
+
+
+class TestPFFTracking:
+    @pytest.fixture
+    def meta_data(self, base_dir) -> str:
+        return base_dir / "files/pff_metadata.csv"
+
+    @pytest.fixture
+    def rosters_meta_data(self, base_dir) -> str:
+        return base_dir / "files/pff_rosters.csv"
+
+    @pytest.fixture
+    def raw_data(self, base_dir) -> str:
+        return base_dir / "files/pff_10517.jsonl.bz2"
+
+    @pytest.fixture
+    def raw_data_1(self, base_dir) -> str:
```

Comment:
The fixture name should be descriptive. How does it differ from `raw_data`? Alternatively, add a docstring.